feature/bgp/policybase/otg_tests/import_export_multi/README.md

```diff
@@ -0,0 +1,207 @@
+# RT-7.11: BGP Policy - Import/Export Policy Action Using Multiple Criteria
+
+## Summary
+
+The purpose of this test is to verify a combination of bgp conditions using matching and policy nesting as well as and actions in a single BGP import policy.  Additional combinations may be added in the future as subtests.
+
+## Testbed type
+
+* [2 port ATE to DUT](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Testbed common configuration
+
+* Testbed configuration - Setup eBGP sessions and prefixes
+  * Generate config for 2 DUT and ATE ports where
+    * DUT port 1 connects to ATE port 1.
+    * DUT port 2 connects to ATE port 2.
+  * Configure ATE port 1 with an external type BGP session to DUT port 1
+    * DUT ASN 65000
+    * ATE port 1 ASN 65100
+    * ATE port 2 ASN 65200
+    * Advertise ipv4 and ipv6 prefixes from ATE port 1 to DUT port 1 using
+      the following communities:
+    * prefix-set-1 with 2 ipv4 and 2 ipv6 routes with communities [ "10:1" ]
+    * prefix-set-2 with 2 ipv4 and 2 ipv6 routes with communities [ "20:1" ]
+    * prefix-set-3 with 2 ipv4 and 2 ipv6 routes with communities [ "30:1" ]
+    * prefix-set-4 with 2 ipv4 and 2 ipv6 routes with communities [ "20:2", "30:3" ]
+    * prefix-set-5 with 2 ipv4 and 2 ipv6 routes with communities [ "40:1" ]
+    * prefix-set-6 with 2 ipv4 and 2 ipv6 routes with communities [ "50:1" ]
+    * Configure accept_route policy and apply as an export policy on the DUT
+      eBGP session to ATE port 2.
```

Comment:
This is a subtle point because we can make this work with or without `default-(import|export)-policy` paths.

I avoided requiring them because it is not clear that vendors consistently support changing their BGP import/export policy defaults while there is 100% support for a policy statement which accepts/rejects all routes following all the match/action conditions.  Writing policy this way avoids the need for deviation and is semantically the equivalent to relying on a default. 

An analysis of vendor support:
* [junos supports changing default bgp import/export policy](https://www.juniper.net/documentation/us/en/software/junos/bgp/topics/topic-map/basic-routing-policies.html#id-example-overriding-the-default-bgp-routing-policy-on-ptx-series-packet-transport)
* [Cisco 's RPL has a command bgp unsafe-ebgp-policy](https://www.cisco.com/c/dam/en_us/training-events/le31/le46/cln/pdf/webinar_slides/RPL-Webinar-Slides-CLN.pdf) mentioned in some community discussions and presentations presumably as a way to import/export all eBGP learned routes, but it not present in [recent documentation for IOS XR](https://www.cisco.com/c/en/us/td/docs/iosxr/cisco8000/bgp/710x/b-bgp-cg-8k-710x/implementing-bgp.html) or even in [old documentation](https://www.cisco.com/c/en/us/td/docs/routers/xr12000/software/xr12k_r3-9/routing/configuration/guide/xr12krc39_chapter1.html).
* [Arista EOS bgp missing-policy](https://www.arista.com/en/support/toi/eos-4-18-0f/13853-bgp-missing-policy-action) allows configuration of a default import/export BGP policy, but appears to only apply when there is no policy present.  If a policy is present (our use case) then the default policy result is presumably to deny routes.
* [Nokia SROS - ebgp-default-reject-policy ](https://infocenter.nokia.com/public/7750SR222R1A/index.jsp?topic=%2Fcom.nokia.Unicast_Guide%2Fdefault_externa-ai9exj5ykx.html)  command allows overriding RFC8212 behavior for eBGP sessions.  I could not find SRLinux documentation on this topic, just a product overview guide saying srlinux is compliant with RFC8212.

As you point out, the downside of not using/having a configurable `default-(import|export)-policy` is  it requires always generating an accept/reject action at the end of every policy to ensure we get the behavior we want consistently across vendors.    If a mistake is made, all vendors appear to honor RFC8212 making  import and export a default deny for eBGP session.  (note it is not clear there is consistent behavior for iBGP across vendors either.  iBGP default policy result is not covered by  RFC8212)

So do we want to write consistent policy without `default-(import|export)-policy`, or do we prefer to attempt to configure `default-(import|export)-policy` and implement deviations for vendors who don't support this?  If we want to require default, then deviations will need to be introduced which may not go away anytime soon or ever.  Based on my analysis above, I'd estimate that 2 out of 3 vendors might need deviation with various caveats.  

If you still want the test to require default import/export after considering this, let me know, I'll make the change.